### PR TITLE
Fix redirecting links in README

### PR DESCRIPTION
**Pull request by JamesSkemp**

### CODE REVIEW

**Comment by JamesSkemp** [comment]
Per https://github.com/nvie/gitflow/pull/6450#issuecomment-1585688851 recreating here.

Looks like the Wiki/FAQ still needs to be setup on this fork. If the other repo is dead you may also want to update the issues link.

Happy to update within this PR, but wanted to keep the spirit of it the same as the original PR.

File: README.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ blog post"). This fork adds functionality not added to the original branch.
...
 
-[http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/)
+[Using git-flow to automate your git branching workflow](https://jeffkreeftmeijer.com/git-flow/)
 
 Or have a look at one of these screen casts:
 
-* [How to use a scalable Git branching model called git-flow](http://buildamodule.com/video/change-management-and-version-control-deploying-releases-features-and-fixes-with-git-how-to-use-a-scalable-git-branching-model-called-gitflow) (by Build a Module)
-* [A short introduction to git-flow](http://vimeo.com/16018419) (by Mark Derricutt)
-* [On the path with git-flow](https://vimeo.com/codesherpas/on-the-path-gitflow) (by Dave Bock)
+* [How to use a scalable Git branching model called git-flow](https://buildamodule.com/index.php?q=video/change-management-and-version-control-deploying-releases-features-and-fixes-with-git-how-to-use-a-scalable-git-branching-model-called-gitflow) (by Build a Module)
```
Comment (ChrisJStone) [inline README.md]: Build a module has been taken down. The videos are over on youtube now. I believe this is the video in question. https://www.youtube.com/watch?v=-8DgENtFWSU

**Review by ChrisJStone — COMMENTED** [review]
Other then the one video link for bulidamodule looks good.

**Comment by ChrisJStone** [comment]
> Looks like the Wiki/FAQ still needs to be setup on this fork. If the other repo is dead you may also want to update the issues link.

Thank you for pointing that out. I'll start getting the Wiki/FAQ setup and get the issue links updated in the readme

**Pull request merged**